lib/inertia/ssr.ex

```diff
@@ -45,7 +48,18 @@ defmodule Inertia.SSR do
   @doc false
   def call(page) do
     module = GenServer.call(Config, :module)
-    NodeJS.call({module, :render}, [page], name: supervisor_name(), binary: true)
+    esm = GenServer.call(Config, :esm)
+    module = if esm do
+      "#{module}.js?q=#{System.unique_integer()}"
```

Comment:
When importing an ESM if it's not from a node package it requires specify the extension.

Reply:
The `q=` is a cache busting for refreshing re-generated `ssr.js` file when it is requested

Reply:
I happened to notice this comment about memory leaking: https://github.com/revelrylabs/elixir-nodejs/pull/84/files#diff-9a4833cd359a3fc7ab8667feb826874b540e3aa79773415b995eda8c22a55773R28-R31

I wonder if the cache busting should be optional? Presumably we wouldn't want to bust cache in production.

The `require.cache` behavior is automatically conditional based on`NODE_ENV` (https://github.com/revelrylabs/elixir-nodejs/pull/84/files#diff-9a4833cd359a3fc7ab8667feb826874b540e3aa79773415b995eda8c22a55773R14-R19), so we could perhaps emulate that behavior for ESM caching in our logic.

Reply:
You're right, how would you name this option?